examples/synthesis.md

```diff
@@ -0,0 +1,228 @@
+monomer = cript.Ingredient(keyword=["monomer"], material=styrene, quantity=[monomer_qty])
+quench = cript.Ingredient(keyword=["quench"], material=butanol, quantity=[quench_qty])
+workup = cript.Ingredient(keyword=["workup"], material=methanol, quantity=[workup_qty])
+```
+
+Finally, we can add the `Ingredient` nodes to the `Process` node.
+
+```python
+process.ingredient += [initiator, solvent, monomer, quench, workup]
+```
+
+# Add Conditions to the Process
+
+Its possible that our `Process` was carried out under specific physical conditions. We can codify this by adding `Condition` nodes to the process.
+
+```python
+temp = cript.Condition(key="temperature", type="value", value=25, unit="celsius")
+time = cript.Condition(key="time_duration", type="value", value=60, unit="min")
+process.condition = [temp, time]
+```
+
+# Add a Property to a Process
+
+We may also want to associate our process with certain properties. We can do this by adding `Property` nodes to the process.
+
+```python
+yield_mass = cript.Property(key="yield_mass", type="number", value=47e-5, unit="kilogram", method="scale")
+process.property += [yield_mass]
+```
+
+# Create a Material node (process product)
+
+Along with input `Ingredients`, our `Process` may also produce product materials.
+
+First, let's create the `Material` that will serve as our product. We give the material a `name` attribute and add it to our `Project`.
+
+```python
+polystyrene = cript.Material(name="polystyrene", identifiers=[])
+project.material += [polystyrene]
+```
+
+Let's add some `Identifier`s to the material to make it easier to identify and search.
+
+```python
+# create a name identifier
+polystyrene.identifiers += [{"names": ["poly(styrene)", "poly(vinylbenzene)"]}]
+
+# create a BigSMILES identifier
+polystyrene.identifiers += [{"bigsmiles": "[H]{[>][<]C(C[>])c1ccccc1[<]}C(C)CC"}]
+# create a chemical repeat unit identifier
+polystyrene.identifiers += [{"chem_repeat": ["C8H8"]}]
+```
+
+Next, we'll add some `Property` nodes to the `Material`, which represent its physical or virtual (in the case of a simulated material) properties.
+
+```python
+# create a phase property
+phase = cript.Property(key="phase", value="solid", type="none", unit=None)
+# create a color property
+color = cript.Property(key="color", value="white", type="none", unit=None)
+
+# add the properties to the material
+polystyrene.property += [phase, color]
+```
+
+**Congratulations!** You've just created a process that represents the polymerization reaction of Polystyrene, starting with a set of input ingredients in various quantities, and ending with a new polymer with specific identifiers and physical properties.
+
+Now we can save the project to CRIPT via the api object.
+
+```python
+project.validate()
+print(project.get_json(indent=2).json)
+# api.save(project)
+```
+
+```python
+# Don't forget to disconnect once everything is done
```

Comment:
I really like this comment for some reason. It just makes it feel so human and nice!

Reply:
awesome :)